packages/web-components/src/presence-badge/README.md

```diff
@@ -0,0 +1,117 @@
+---
+id: card
+title: fluent-card
+sidebar_label: card
+custom_edit_url: https://github.com/microsoft/fluentui/edit/master/packages/web-components/src/card/README.md
+---
+
+# fluent-presence-badge
+
+## Overview
+
+A `fluent-presence-badge` is designed to be used with the Avatar control to display a status icon based on the in/out/busy status of an Office user.
+
+## API
+
+Extends FAST Element
+
+_Component Name_
+
+- `fluent-presence-badge`
+
+_Attributes_
+
+- `status` - string value that sets the icon to display: `"available" | "away" | "busy" | "dnd" | "outofoffice" | "offline" | "blocked"`
```

Comment:
`"dnd"` seems to be the only abbreviated value, for consistency should it be `"donotdisturb"`